docs/api-usage/authentication.mdx

```diff
@@ -117,6 +170,12 @@ mutation {
 }
 ```
 
+:::caution
+Keep in mind that you should **not** add the `Authorization` header with an expired access token to this request. It'll result in an error.
```

Comment:
```suggestion
Keep in mind that you should **not** add the `Authorization` header with an expired access token to this request. It will result in an error.
```